examples/sql_bank/main.go

```diff
@@ -49,35 +82,9 @@ func moveMoney(db *sql.DB) {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if _, err = tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
-			log.Fatal(err)
-		}
-		rows, err := tx.Query("SELECT id, balance FROM accounts WHERE id IN ($1, $2)", from, to)
```

Comment:
Another reason for keeping the currently single query, it will make it really obvious when I fix #2140.